packages/bloc_test/README.md

```diff
@@ -194,3 +194,7 @@ test('emits [1] when CounterEvent.increment is added', () async {
 ## Maintainers
 
 - [Felix Angelov](https://github.com/felangel)
+
+## Supporters
+
+[<img src="https://raw.githubusercontent.com/felangel/bloc/master/docs/assets/vgv_logo.png" width="120px" />](https://verygood.ventures)
```

Comment:
```suggestion
[<img src="https://raw.githubusercontent.com/felangel/bloc/master/docs/assets/vgv_logo.png" width="120" />](https://verygood.ventures)
```

Reply:
@zepfietje why do you prefer to avoid the units?

Reply:
@felangel https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-width

Reply:
oops! I was thinking of the width/height css styles 😛